package.json

```diff
@@ -18,7 +18,7 @@
     "nightly": "npm run travis:verify",
     "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
     "server:ctr": "node src/server/generateServerKey.js",
-    "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
+    "start": "NODE_ENV=development USEPROXY=true webpack serve --config config/dev.webpack.config.js",
```

Comment:
This will force using the proxy, do you really want it? I guess ading new z`start:proxy` might be a better approach.

Reply:
mmm well i mention in the readme removing that allows all original functionality...  but yeah adding a new script is probs even more kind